components/Slider/src/MDCSlider.m

```diff
@@ -584,7 +582,7 @@ - (void)accessibilityIncrement {
   if (self.enabled) {
     CGFloat range = self.maximumValue - self.minimumValue;
     CGFloat adjustmentAmount = kSliderAccessibilityIncrement * range;
-    if (!self.continuous && self.numberOfDiscreteValues > 1) {
+    if (self.numberOfDiscreteValues > 1) {
```

Comment:
please add a check for if (discrete) here too

Reply:
Done.